test/test.js

```diff
@@ -24,8 +24,8 @@ test('allows to retrieve all browsers on Sauce Labs', (t) => {
 });
 
 test('excludes browsers not available on Sauce Labs', (t) => {
-  sauceBrowsers([{ name: 'foo' }]).then((browsers) => {
-    t.deepEqual(browsers, []);
+  sauceBrowsers([{ name: 'foo' }]).catch((err) => {
+    t.equals(err.message, 'Browser foo is not available');
```

Comment:
Please also add `t.equal(err instanceof Error, true);` for consistency with the other tests.